### PR TITLE
Feature/pedrogarriganogues cv

### PR DESCRIPTION
**Pull request by PedroGarrigaNogues**

### CODE REVIEW

File: staff/pedro-garriganogues/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <dt>Email:</dt>
+        <dt>pedrogarriganogues@outlook.com</dt>
+        <dt>Phone:</dt>
+        <dt>+34 639 701 910</dt>
+    </dl>
+    <hr>
+</head>
+<h2>Profile Summary:</h2>
+<hr>
+<p></p>
```
Comment (msarroca) [inline staff/pedro-garriganogues/cv/index.html]: No text inside

File: staff/pedro-garriganogues/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+<html>
+<head>
+
+    <h1>Pedro Garriga-Nogués</h1>
+    <hr>
+    <dl>
+        <img src="handsome.jpg" alt="Oooopssss">
+        <dt><a href="https://github.com/PedroGarrigaNogues">GitHub </a><a href="https://www.linkedin.com/in/pedro-garriga-nogu%C3%A9s-2ab577a5/">Linkedin</a></dt>
+        <dt>Email:</dt>
+        <dt>pedrogarriganogues@outlook.com</dt>
```
Comment (maryammdot) [inline staff/pedro-garriganogues/cv/index.html]: If you decide to use dl dt and dd tags, you should use some dd dor the explanations of each term. check the info here: https://www.w3schools.com/tags/tag_dt.asp

File: staff/pedro-garriganogues/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <dt>+34 639 701 910</dt>
+    </dl>
+    <hr>
+</head>
+<h2>Profile Summary:</h2>
+<hr>
+<p></p>
+<body>
+    
+<dl>
```
Comment (maryammdot) [inline staff/pedro-garriganogues/cv/index.html]: Maybe instead of <dl> tag, here you could use <section>

**Comment by quinwacca** [comment]
Yep! Move the visual part of the html inside <body> tags! Use this guide:
```javascript
<!DOCTYPE html>
<html>
<head>
  <title>Title of the document</title>
</head>

<body>
The content of the document......
</body>

</html>
``` 


Your code right now:
```javascript
<head>

    <h1>Pedro Garriga-Nogués</h1>
    <hr>
    <dl>
        <img src="handsome.jpg" alt="Oooopssss">
        <dt><a href="https://github.com/PedroGarrigaNogues">GitHub </a><a href="https://www.linkedin.com/in/pedro-garriga-nogu%C3%A9s-2ab577a5/">Linkedin</a></dt>
        <dt>Email:</dt>
        <dt>pedrogarriganogues@outlook.com</dt>
        <dt>Phone:</dt>
        <dt>+34 639 701 910</dt>
    </dl>
    <hr>
</head>

```

File: staff/pedro-garriganogues/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE <!DOCTYPE html>
+<html>
+<head>
```
Comment (quinwacca) [inline staff/pedro-garriganogues/cv/index.html]: Use this template to fix your <head> and <body> tags!
```javasript
<!DOCTYPE html>
<html>
<head>
  <title>Title of the document</title>
</head>

<body>
The content of the document......
</body>

</html>
```

**Pull request merged**